flang/lib/Lower/Allocatable.cpp

```diff
@@ -379,6 +379,9 @@ class AllocateStmtHelper {
   }
 
   void lowerAllocation(const Allocation &alloc) {
+    if (HasCUDAAttr(alloc.getSymbol()))
```

Comment:
don't we need `Fortran::semantics::` here? Obviously, there is probably some using ... spilling here. But unless it is consistently not used in the file, please add it.

Reply:
It somehow works without it but I added the namespaces to be consistent with the rest of the file.